environment_py3_linux.yml

```diff
@@ -28,4 +28,5 @@ dependencies:
   - nbval
   - scikit-learn
   - pykdtree
+  - cartopy
```

Comment:
Why is cartopy now need? That was not the case before, and I suggest not to keep it
```suggestion
```

Reply:
that will make the _new_ test fail that actually tests platecarree plotting